core-bundle/tests/Filesystem/VirtualFilesystemTest.php

```diff
@@ -557,15 +557,13 @@ static function () use (&$handlerInvocationCount) {
         $this->assertInstanceOf(FilesystemItem::class, $fileB);
         $this->assertTrue($fileB->isFile());
 
-        /** @phpstan-ignore-next-line */
```

Comment:
@m-vo I keeps getting stranger. Now PHPStan said _No error to ignore is reported on line 561_ and neither on line 567, but the `@phpstan-ignore-next-line` on line 544 is still necessary. 🤔